src/sherpa_ai/prompts/Base.py

```diff
@@ -0,0 +1,25 @@
+from pydantic import BaseModel
+from typing import List, Dict, Union
+
+
+class TextPrompt(BaseModel):
+    content: str
+
+
+class ChatPrompt(BaseModel):
+    content: List[Dict[str, str]]
+
+
+class Prompt(BaseModel):
+    name: str
+    description: str
+    version: str
+    content: Union[ChatPrompt, TextPrompt]
+    variables: Dict
+    response_format: Dict
```

Comment:
I don't think this is used. Or maybe TextPrompt and ChatPrompt should extend the Prompt class I think